models/evc.py

```diff
@@ -399,6 +414,32 @@ def archive(self):
         """Archive this EVC on deletion."""
         self.archived = True
 
+    def _use_uni_vlan(self, uni: UNI):
+        """Use tags from UNI"""
+        if uni.user_tag is None:
+            return
+        tag = uni.user_tag.value
+        tag_type = uni.user_tag.tag_type.value
+        if isinstance(tag, int):
+            if not uni.interface.use_tags([tag, tag], tag_type):
+                intf = uni.interface.id
+                raise ValueError(f"Tag {tag} is not available in {intf}")
+            uni.interface.notify_interface_tags(self._controller)
+
+    def _disuse_uni_vlan(self, uni: UNI):
```

Comment:
Could you try to use the `make_*_available` naming here too, first time the `disuse` name is showing up. Makes sense, but let's follow a similar naming convention you're already using the core ideally.

Reply:
Changed name to `make_uni_vlan_available()`